packages/vaadin-grid/src/vaadin-grid-column.js

```diff
@@ -548,107 +450,91 @@ export const ColumnBaseMixin = (superClass) =>
     }
 
     /**
-     * Renders either a template or a renderer to the given cells.
-     * Throws an error if both are passed, the template and the renderer.
+     * Renders the content to the given cells using a renderer.
      *
      * @private
      */
-    __renderCellsContent(template, renderer, cells) {
+    __renderCellsContent(renderer, cells) {
       // Skip if the column is hidden or not attached to a grid.
       if (this.hidden || !this._grid) {
         return;
       }
 
-      if (template && renderer) {
-        throw new Error('You should only use either a renderer or a template');
```

Comment:
What happens if you try to use both a template and a custom renderer at the same time now? Maybe template-renderer should throw an error (could be added in a separate PR)?

Reply:
Good point! It doesn't seem difficult to implement. I would suggest adding this in a separate PR right after the current PR is merged.